AK/Debug.h.in

```diff
@@ -470,6 +470,10 @@
 #cmakedefine01 WEBSERVER_DEBUG
```

Comment:
This commit now has Andreas as author was that your intention?
(See https://www.git-tower.com/learn/git/faq/change-author-name-email/ for example if you want to change/fix it)

Reply:
I rebased against master and kinda messed up I think (had to go back and edit the message). I don't mind if the first commit is tagged w/ him as the Author. I can fix it if needed.

Reply:
I mean you wrote the commit so I would say you deserve the credit (and blame :wink:).
I think it's better if it's you in case people have questions etc.